src/components/Button.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import clsx from "clsx";
+import { ButtonProps, ButtonVarient } from "../types/ButtonTypes";
+
+const Button: React.FC<ButtonProps> = ({
+  label,
+  onClick,
+  style,
+  varient,
```

Comment:
I think it is better to give a default variant. If we didn't specify a variant it can take the default one